src/app/core/services/split-expense.service.ts

```
@@ -415,4 +417,104 @@ export class SplitExpenseService {
+    const threshold = this.getThresholdTolerance(precision);
+    const roundedTargetAmount = Number.parseFloat(targetAmount.toFixed(precision));
+    let totalRoundedAmount = 0;
+
+    splitExpensesFormArray.controls.forEach((control) => {
+      const currentAmount = (control.get('amount')?.value as number) || 0;
+      const roundedAmount = Number.parseFloat(currentAmount.toFixed(precision));
+
+      control.patchValue(
+        {
+          amount: roundedAmount,
+          percentage: targetAmount > 0 ? parseFloat(((roundedAmount / targetAmount) * 100).toFixed(3)) : 0,
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)